Formula/lib/libaec.rb

```diff
@@ -19,21 +20,18 @@ class Libaec < Formula
 
   depends_on "cmake" => :build
 
-  # These may have been linked by `szip` before keg_only change
-  link_overwrite "include/szlib.h"
-  link_overwrite "lib/libsz.a"
-  link_overwrite "lib/libsz.dylib"
-  link_overwrite "lib/libsz.2.dylib"
-  link_overwrite "lib/libsz.so"
-  link_overwrite "lib/libsz.so.2"
```

Comment:
Should probably keep this for users who haven't run `brew upgrade` in a long time.